oyatplum/react-todo/src/component/todoitem.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+const TodoItem = (props) => {
+    const deleteItem = () => { //상속 이용하여 투두 제거
+        var tmp = props.todoList.filter((element)=> element.id !== props.itemId);
+        props.setTodoList(tmp);
```

Comment:
```suggestion
        setTodoList(todoList.filter((element) => element.id !== itemId));
```
var를 지양합시다...! setState에 인자로 바로 넣으면 깔끔하겠죠??
추가로 구조분해할당으로 props를 가져왔기 때문에 곧바로 `itemId` 라는 변수를 쓸 수 있습니다